config/config.json

```diff
@@ -1,15 +1,15 @@
 {
     "parameters": [
-        {"name": "login", "type": "string", "default": "127.0.0.1"},
+        {"name": "login", "type": "str", "default": "127.0.0.1"},
```

Comment:
yes probably but I would like some time very generic so, if we work with str in python why not to use the type format in the json file.

Reply:
and in the other section where we describe input parameters for commands we then use the java script type name ? (because frontend needs to use it )...
and in the json schema to validate parameters and responses we use json schema type names?